pkg/services/x509/x509_validator_test_utils.go

```diff
@@ -0,0 +1,133 @@
+package x509
```

Comment:
I'd give this file _test postfix so it doesn't get analyzed by our code analyses/coverage tooling

(or put it in /test)